[2024-08-07T20:33:47.525+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-07T20:33:47.966+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: space_exploration.satellite_response scheduled__2023-02-22T00:00:00+00:00 [queued]>
[2024-08-07T20:33:48.003+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: space_exploration.satellite_response scheduled__2023-02-22T00:00:00+00:00 [queued]>
[2024-08-07T20:33:48.011+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-07T20:33:48.121+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): satellite_response> on 2023-02-22 00:00:00+00:00
[2024-08-07T20:33:48.136+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=378) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-07T20:33:48.141+0000] {standard_task_runner.py:64} INFO - Started process 407 to run task
[2024-08-07T20:33:48.155+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'space_exploration', 'satellite_response', 'scheduled__2023-02-22T00:00:00+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/***-space-exploration.py', '--cfg-path', '/tmp/tmptcr81e5t']
[2024-08-07T20:33:48.177+0000] {standard_task_runner.py:91} INFO - Job 102: Subtask satellite_response
[2024-08-07T20:33:48.329+0000] {task_command.py:426} INFO - Running <TaskInstance: space_exploration.satellite_response scheduled__2023-02-22T00:00:00+00:00 [running]> on host 13935ade345d
[2024-08-07T20:33:48.634+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='space_exploration' AIRFLOW_CTX_TASK_ID='satellite_response' AIRFLOW_CTX_EXECUTION_DATE='2023-02-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-02-22T00:00:00+00:00'
[2024-08-07T20:33:48.638+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-07T20:33:48.679+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-07T20:33:48.680+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-07T20:33:48.701+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=space_exploration, task_id=satellite_response, run_id=scheduled__2023-02-22T00:00:00+00:00, execution_date=20230222T000000, start_date=20240807T203347, end_date=20240807T203348
[2024-08-07T20:33:48.783+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-07T20:33:48.887+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-07T20:33:48.903+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
